packages/api-sdk/src/medical/models/patient.ts

```diff
@@ -10,6 +10,7 @@ export type PatientUpdate = z.infer<typeof patientUpdateSchema>;
 
 export const patientSchema = patientUpdateSchema.extend({
   facilityIds: z.array(z.string()),
+  externalId: z.string().optional(),
```

Comment:
we should update the createPatient function in the sdk too no?

Reply:
No because this is just to update the `patientSchema` its not updating the updateSchema

Reply:
But we updated the createPatient function in create-patient.ts. And the POST patient route. I might just be confused though, but what I am saying is that in metriport.ts in the client in the sdk, we should update it so you can call it with an external id

Reply:
I see you what you mean, for consistency i decided to not it include it in the update because we dont have the option to update the facility param either. Now that i think about it though it could be down to issue updating that with CW so ill include that in the update as well

Reply:
Even if we don't want to allow for update, I agree it should be possible to set it during creation when using the SDK.

Reply:
I misunderstood @jonahkaye but yea i updated the sdks